io_mesh_w3d/common/structs/mesh_structs/shader_material.py

```diff
@@ -163,32 +163,25 @@ def create(self, parent):
             xml_value = create_node(xml_constant, 'Value')
             xml_value.text = self.value
 
-        elif self.type == 2:
+        elif self.type > 1 and self.type < 6:
```

Comment:
Couldn't you use constants instead of straight integer numbers? When reading through this noone knows what these types are

Reply:
yea, but > FLOAT and < BOOL wouldnt make much sense either